crypto/engine/eng_lib.c

```diff
@@ -149,8 +149,10 @@ void engine_cleanup_add_last(ENGINE_CLEANUP_CB *cb)
     if (!int_cleanup_check(1))
         return;
     item = int_cleanup_item(cb);
-    if (item)
-        sk_ENGINE_CLEANUP_ITEM_push(cleanup_stack, item);
+    if (item) {
```

Comment:
Should that be tested against NULL?  Not worth delaying this PR if it will take awhile to get around to fixing that nit.